### PR TITLE
Setup dashboards for patterns (java, nginx) on EKS

### PR DESCRIPTION
**Pull request by bonclay7**
*Issue #, if available:*

*Description of changes:*

- Create CloudWatch and X-Ray data sources on Grafana
- Split dashboards manifest (infra, java, nginx, amp)  so each module can request a dashboard to be created based on customer configuration

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: artifacts/grafana-operator-manifests/kustomization.yaml [hunk 1]
```diff
@@ -3,4 +3,9 @@ kind: Kustomization
 resources:
   - amg_grafana-identity.yaml
   - amg_grafana-amp-datasource.yaml
-  - amg_grafana-dashboard.yaml
+  - amg_grafana-cw-datasource.yaml
+  - amg_grafana-xray-datasource.yaml
+  - amg_grafana-dashboards-amp.yaml
```
Comment (elamaran11) [inline artifacts/grafana-operator-manifests/kustomization.yaml:8]: If you do this. All yamls will be deployed at one short even if the customer is not looking to java or ngnix monitoring. I have design in mind for this. You should have separate kustomization for this and have separate folder which can be sync'd by kustomization for individual monitoring needs. We should not club all under one `kustomization`.

**Review by elamaran11 — COMMENTED** [review]
@bonclay7 I love the approach of adding `nginx`, `java` monitoring modules to the accelerator solution. We should have different design to align to separation of concerns. If you do what you have in this PR, everything gets deployed even if customer is looking for infra monitoring only. I recommend having separate `kustomization` resource for each individual monitoring needs like Java, nginx and move java and nginx manifests to seprate folder and have that render in the new Kustomization so we have separation of concerns and single responsibility principle. We can talk about it if you want. We can design it efficiently as a pattern for others to folllow.

File: artifacts/grafana-operator-manifests/kustomization.yaml (hunk 1 above)
Comment (bonclay7) [reply artifacts/grafana-operator-manifests/kustomization.yaml:8]: Awesome! TBH I didn't know how to separate the resources creation, I'll give a go at what you say if I understood well. But my goal is exactly what you say, deploy what customer wants and not everything

File: artifacts/grafana-operator-manifests/amp/kustomization.yaml [hunk 2]
```diff
@@ -2,5 +2,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - amg_grafana-identity.yaml
```
Comment (elamaran11) [inline artifacts/grafana-operator-manifests/amp/kustomization.yaml:4]: Remove Grafana Identity

File: artifacts/grafana-operator-manifests/eks/java/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - amg_grafana-identity.yaml
```
Comment (elamaran11) [inline artifacts/grafana-operator-manifests/eks/java/kustomization.yaml:4]: Remove Grafana Identity

**Review by elamaran11 — COMMENTED** [review]
Have one version of Grafana Identity

File: artifacts/grafana-operator-manifests/amp/amg_grafana-cw-datasource.yaml [hunk 4]
```diff
@@ -1,13 +1,13 @@
 apiVersion: grafana.integreatly.org/v1beta1
```
Comment (elamaran11) [inline artifacts/grafana-operator-manifests/amp/amg_grafana-cw-datasource.yaml:1]: This file is duplicate. Keep all DS in infra

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**